tests/phpunit/tests/functions/getThePostsNavigation.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Tests get_the_posts_navigation function
+ *
+ * @ticket 55751
+ * @since 6.1
+ *
+ * @covers ::get_the_posts_navigation
+ */
+class Tests_Functions_GetThePostsNavigation extends WP_UnitTestCase {
+
+	/**
+	 * Create posts for the tests.
+	 *
+	 * @param WP_UnitTest_Factory $factory
+	 */
+	public static function wpsetupBeforeClass( WP_UnitTest_Factory $factory ) {
+		$factory->post->create_many( 3 );
+	}
+
+	/**
+	 * Data provider method for testing `get_the_posts_navigation()`.
+	 *
+	 * @return array
+	 */
+	public function data_get_the_posts_navigation() {
+		return array(
+			array( 1, 1, true, false, false ),
```

Comment:
just as a note, you can use the array keys for the test data provider if you want to be more clear and specific about what is each value ;) cheers